### PR TITLE
mrinal/command

### PR DESCRIPTION
**Pull request by mrinalwadhwa**
- refactor(rust): enroll, project and space commands
- feat(rust): improve ockam command help
- feat(rust): improve ockam node command help

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/cloud/mod.rs [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
+    /// Enroll with Ockam Cloud
+    #[clap(display_order = 900)]
+    Enroll(EnrollCommand),
+
+    /// Create, update and delete projects in Ockam Cloud
+    #[clap(display_order = 900)]
+    Project(ProjectCommand),
+
+    /// Create, update and delete spaces in Ockam Cloud
+    #[clap(display_order = 900)]
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/cloud/mod.rs:28]: Shouldn't the `display_order` be incremental? Or does it behave in a certain way if you assign the same order to multiple commands?

**Review by adrianbenavides — COMMENTED** [review]
Love it! I think it all looks more homogeneous.

File: implementations/rust/ockam/ockam_command/src/cloud/mod.rs (hunk 1 above)
Comment (mrinalwadhwa) [reply implementations/rust/ockam/ockam_command/src/cloud/mod.rs:28]: Things with the same `display_order` are displayed alphabetically.
The default `display_order` value is 999, so the `help` subcommand is at 999 ... I chose 900 as our default so our subcommands show alphabetically before the `help` command.
https://docs.rs/clap/latest/clap/struct.Arg.html#method.display_order

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/cloud/mod.rs:28]: Ohh, good to know, thanks!

**Pull request merged**